src/Conversion/ONNXToKrnl/ConvertONNXToKrnl.cpp

```diff
@@ -55,7 +55,7 @@ struct FrontendToKrnlLoweringPass
   // Make sure that we have a valid default constructor and copy
   // constructor to make sure that the options are initialized properly.
   FrontendToKrnlLoweringPass() = default;
-  FrontendToKrnlLoweringPass(const FrontendToKrnlLoweringPass &pass) {}
+  FrontendToKrnlLoweringPass(const FrontendToKrnlLoweringPass &pass){};
```

Comment:
That one is strange to me, though the ";" was needed for declarations, and was not needed when giving the code.